tempto-core/src/main/java/io/prestosql/tempto/fulfillment/table/TableDefinitionsRepository.java

```diff
@@ -108,12 +67,13 @@ public static TableDefinition tableDefinition(TableHandle tableHandle)
     public static TableDefinitionsRepository tableDefinitionsRepository()
     {
         if (TABLE_DEFINITIONS_REPOSITORY == null) {
-            TABLE_DEFINITIONS_REPOSITORY = new TableDefinitionsRepository(ImmutableList.<TableDefinition>builder()
-                    .addAll(BUILTIN_TABLE_DEFINITIONS)
-                    .addAll(ADDITIONAL_TABLE_DEFINITIONS.get())
-                    // TODO: since TestNG has no listener that can be run before tests factory, this has to be initialized here
-                    .addAll(getAllConventionBasedTableDefinitions())
-                    .build());
+            List<TemptoPlugin> plugins = ImmutableList.copyOf(ServiceLoader.load(TemptoPlugin.class).iterator());
```

Comment:
I would write some dedicated method to load plugins. `TemptoPlugins.load()`?

Reply:
Ideally the plugins should be loaded once, but I don't see a way to properly keep state in tempto suite.
for now, i'm leaving this one-liner as-is.